.circleci/config.yml

```diff
@@ -0,0 +1,18 @@
+version: 2.1
+orbs:
+  aws-cli: circleci/aws-cli@0.1.16
+
+jobs:
+  build-and-push:
+    executor: machine
+    steps:
+      - checkout
+      - aws-cli/setup:
```

Comment:
What the step is doing?

Reply:
it installs the AWS CLI and configured it.